### DISCUSSION TITLE
Cannot authenticate to graphql-explorer – "The change you wanted was rejected"

### DISCUSSION THREAD

**Discussion by kevmoo** [community/community#24580] [Apps, API and Webhooks]
<p><img src="https://user-images.githubusercontent.com/17034/181085315-42cc108e-10ff-4b20-8912-b02ebeae5ece.png" alt="Screen Shot 2020-01-30 at 1.45.49 PM.png" data-base62-sha1="tqDZUMyKuxzwgwXDFJtuG23yb2Y" width="587" height="220"></p>
<p>This is what I see when I click “Sign in with GitHub” on <a href="https://developer.github.com/v4/explorer/" rel="nofollow noopener">https://developer.github.com/v4/explorer/</a></p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Hi <a class="mention" href="/kevmoo">@kevmoo</a>,</p>
<p>Thank you for sharing!  Are you able to sign in succesfully in a different browser or in incognito/private mode within the browser?  If those options are not helpful can you please provide me with the operating system and browser type and version numbers?</p>

**Comment by kevmoo** [comment]
<p>Works fine in Firefox. Still getting it with Chrome 79.0.3945.130  on MacOS.</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Hi <a class="mention" href="/kevmoo">@kevmoo</a>!</p>
<p>Thank you for sharing the browser and OS details!  I’ve shared that information with our team and they are looking into this.</p>

**Comment by NghiaTranUIT** [comment]
<p>FYI: I encountered the same bug on Safari (Version 13.0.5 (15608.5.11)), macOS 10.15.3.</p>

**Comment by yakov116** [comment]
<p>Having the same issue now </p>

**Comment by BryanAM** [comment]
<p>Also have the same issue. I am able to login with Chrome.</p>
<ol>
<li>Safari + Safari Private Browsing</li>
</ol>
<ul>
<li>v13.0.5</li>
<li>MacOS Catilina 10.15.3</li>
</ul>

**Comment by igo** [comment]
<blockquote>
<p><a class="mention" href="/nyahbhinghiprincess">@nyahbhinghiprincess</a> wrote:</p>
</blockquote>
<blockquote>
<p>Hi <a class="mention" href="/kevmoo">@kevmoo</a>!</p>
</blockquote>
<blockquote>
<p>Thank you for sharing the browser and OS details!  I’ve shared that information with our team and they are looking into this.</p>
</blockquote>
<p>Hi <a class="mention" href="/nyahbhinghiprincess">@nyahbhinghiprincess</a>, any news? Cannot login nor in Chrome (both Mac or Windows) nor Safari.</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Thank you for checking in <a class="mention" href="/igo">@igo</a> !  There isn’t a resolution yet.<br>
The team is currently investigating the issue and we apologize for the inconvenience during this time.  In the meantime, if possible, you may want to try out one of our other <a href="https://help.github.com/en/github/getting-started-with-github/supported-browsers" rel="nofollow noopener">supported browsers </a>.</p>

**Comment by yakov116** [comment]
<p>This has to get fixed I am trying to build something and its not normal!!</p>

**Comment by yakov116** [comment]
<pre><code>devAssert.mjs:5 Uncaught (in promise) Error: Invalid or incomplete introspection result. Ensure that you are passing "data" property of introspection response and no "errors" was returned alongside: null
    at r (devAssert.mjs:5)
    at Ze (buildClientSchema.mjs:27)
    at GraphiQLWithExtensions.js:197
r @ devAssert.mjs:5
Ze @ buildClientSchema.mjs:27
(anonymous) @ GraphiQLWithExtensions.js:197
Promise.then (async)
value @ GraphiQLWithExtensions.js:189
e.notifyAll @ CallbackQueue.js:80
close @ ReactReconcileTransaction.js:85
closeAll @ Transaction.js:212
perform @ Transaction.js:154
N @ ReactMount.js:148
perform @ Transaction.js:141
batchedUpdates @ ReactDefaultBatchingStrategy.js:55
batchedUpdates @ ReactUpdates.js:94
_renderNewRootComponent @ ReactMount.js:351
_renderSubtreeIntoContainer @ ReactMount.js:435
render @ ReactMount.js:458
x @ graphql-explorer.js:137
(anonymous) @ graphql-explorer.js:49
l @ application-5cf251716828380245f62b745a4b525cab9929c9a8fc93aef64f431ff9c33ff7.js:25
fireWith @ application-5cf251716828380245f62b745a4b525cab9929c9a8fc93aef64f431ff9c33ff7.js:25
ready @ application-5cf251716828380245f62b745a4b525cab9929c9a8fc93aef64f431ff9c33ff7.js:25
s @ application-5cf251716828380245f62b745a4b525cab9929c9a8fc93aef64f431ff9c33ff7.js:14
</code></pre>

**Comment by yakov116** [comment]
<p>Any updates??</p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hi <a class="mention" href="/kevmoo">@kevmoo</a> and <a class="mention" href="/igo">@igo</a> I see that our team has been working on some changes.  Are you still noticing the same behavior?</p>

**Comment by kevmoo** [comment]
<p>I’ll look again!</p>

**Comment by kevmoo** [comment]
<p>I was able to authenticate again! All good! Thank you!</p>

**Comment by igo** [comment]
<p>yep, it’s working again</p>

**Comment by yakov116** [comment]
<p>Its working thanks!!</p>
